src/types/AdsConsent.interface.ts

```diff
@@ -0,0 +1,365 @@
+   * Mobile SDKs and saved.
+   *
+   * If the user is outside of the EEA, the request form will error.
+   *
+   * #### Example
+   *
+   * ```js
+   * import { AdsConsent, AdsConsentStatus } from '@invertase/react-native-google-ads';
+   *
+   * async function requestConsent() {
+   *   const consent = await AdsConsent.requestInfoUpdate(['pub-6189033257628554']);
+   *
+   *   // Check if user requires consent
+   *   if (consent.isRequestLocationInEeaOrUnknown && consent.status === AdsConsentStatus.UNKNOWN) {
+   *     // Show a Google-rendered form
+   *     const result = await AdsConsent.showForm({
+   *       privacyPolicy: 'https://invertase.io/privacy-policy',
+   *       withPersonalizedAds: true,
+   *       withNonPersonalizedAds: true,
+   *       withAdFree: true,
+   *     });
+   *
+   *     console.log('User accepted personalized: ', result.status === AdsConsentStatus.PERSONALIZED);
+   *     console.log('User accepted non-personalized: ', result.status === AdsConsentStatus.NON_PERSONALIZED);
+   *     console.log('User prefers Ad Free version of app: ', result.userPrefersAdFree);
+   *   }
+   * }
+   *
+   * ```
+   *
+   * @param options An AdsConsentFormOptions interface to control the Google-rendered form.
+   */
+  showForm(options: AdsConsentFormOptions): Promise<AdsConsentFormResult>;
+
+  /**
+   * Returns a list of ad providers currently in use for the given Google Mobile Ads App ID.
+   *
+   * If requesting consent from the user via your own method, this list of ad providers must be shown to the user
+   * for them to accept consent.
+   *
+   * #### Example
+   *
+   * ```js
+   * import { AdsConsent } from '@invertase/react-native-google-ads';
+   *
+   * const providers = await AdsConsent.getAdProviders();
+   * ```
+   */
+  getAdProviders(): Promise<AdProvider[]>;
+
+  /**
+   * Sets the debug geography to locally test consent.
+   *
+   * If debugging on an emulator (where location cannot be determined) or outside of the EEA,
+   * it is possible set your own location to test how your app handles different scenarios.
+   *
+   * If using a real device, ensure you have set it as a test device via `addTestDevice()` otherwise this method will have
+   * no effect.
+   *
+   * #### Example
+   *
+   * ```js
+   * import { AdsConsent, AdsConsentDebugGeography } from '@invertase/react-native-google-ads';
+   *
+   * // Set disabled
+   * await AdsConsentDebugGeography.setDebugGeography(AdsConsentDebugGeography.DISABLED);
+   *
+   * // Set within EEA
+   * await AdsConsentDebugGeography.setDebugGeography(AdsConsentDebugGeography.EEA);
+   *
+   * // Set outside EEA
+   * await AdsConsentDebugGeography.setDebugGeography(AdsConsentDebugGeography.NOT_EEA);
+   * ```
+   *
+   * @param geography The debug geography location.
+   */
+  setDebugGeography(
+    geography:
+      | AdsConsentDebugGeography['DISABLED']
```

Comment:
doesn't indexing in via string defeat the type, why not `AdsConsentDebugGeography.DISABLED` ?